ui/app/instrumentation.client.ts

```diff
@@ -0,0 +1,75 @@
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+
+  // Environment configuration
+  environment: process.env.NEXT_PUBLIC_SENTRY_ENVIRONMENT || "development",
```

Comment:
```suggestion
  environment: process.env.NEXT_PUBLIC_SENTRY_ENVIRONMENT || "local",
```

I prefer this for local testing.